### PR TITLE
fixed cohort box in sign up

### PR DESCRIPTION
**Pull request by jdowns525**

### CODE REVIEW

File: app/views/user_authentication/sign_up.html.erb [hunk 1]
```diff
@@ -1,58 +1,74 @@
...
-    <label for="last_name_box" class="form-label">Select Cohort</label>
-    <select class="form-select" aria-label="Default select example">
-    <option selected>Select</option>
-    <option value="1">1 - September 21st </option>
-    <option value="2">2 - January 30th </option>
-    <option value="3">3 - February 27th</option>
-    </select>
+    <label for="cohort_box" class="form-label">Select Cohort</label>
+
+    <input id="cohort_box" class="form-control border-green" name="query_cohort_id" placeholder="Choose a cohort..." type="cohort">
```
Comment (MarcusJMyrick) [inline app/views/user_authentication/sign_up.html.erb:29]: Make this a drop down menu of the cohort IDs so they can be choosen when signing up

Comment (jdowns525) [reply app/views/user_authentication/sign_up.html.erb]: fixed

File: app/views/user_authentication/sign_up.html.erb [hunk 2]
```diff
@@ -1,58 +1,80 @@
...
-    <option selected>Select</option>
-    <option value="1">1 - September 21st </option>
-    <option value="2">2 - January 30th </option>
-    <option value="3">3 - February 27th</option>
+  <div class="mb-4">
+    <select name="query_cohort_id" id="cohort-select" class="form-select" aria-label="Default select example">
+      <option value=""> Select Cohort</option>
+      <option value="1">1 - September 21st </option>
+      <option value="2">2 - January 30th </option>
+      <option value="3">3 - February 27th</option>
```
Comment (MarcusJMyrick) [inline app/views/user_authentication/sign_up.html.erb:31]: we can make this dynamic later but its working so far

**Review by MarcusJMyrick — APPROVED** [review]
Good just make this cohort drop down Dynamic in the future

**Pull request merged**